### ISSUE TITLE
bug: No module named ruyaml

### ISSUE THREAD

**Issue by lyz-code** [pycontribs/ruyaml#26]
Hi, I'm having issues installing ruyaml.

```bash
$: virtualenv -p `which python3.7` env
$: source env/bin/activate

$: pip install ruyaml
Collecting ruyaml
  Using cached ruyaml-0.19.2-py2.py3-none-any.whl (99 kB)
Requirement already satisfied: setuptools>=39.0 in ./env/lib/python3.7/site-packages (from ruyaml) (50.3.2)
Collecting distro>=1.3.0
  Using cached distro-1.5.0-py2.py3-none-any.whl (18 kB)
Installing collected packages: distro, ruyaml
Successfully installed distro-1.5.0 ruyaml-0.19.2

$: python -c "import ruyaml as yaml"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'ruyaml'
```

Could you point me on the right direction?

Thanks for the fork by the way :)

**Comment by ejether** [comment]
I've hit the same issue. I found that the current tip of master does allow you to import `ruaml` but we need a release so we can use it as a dependency 😬  (https://github.com/pycontribs/ruyaml/tree/903c66f8e34a171fd470a9db980cc8f087e1db62)


 💯 thanks for the fork!

**Comment by sudermanjr** [comment]
@ssbarnea  Any chance of getting a release out soon? I would love to start using this as I'm hoping it will work better with pyinstaller in one of our projects.

**Comment by ssbarnea** [comment]
Fixed, we have 0.20.0 on https://pypi.org/project/ruyaml/
